### PR TITLE
Upgrade to react native 0.60.0

### PR DESCRIPTION
**Pull request by ryanlntn**
* Upgrades to react native 0.60.0
* Opts out of autolinking for now
* Uses jetifier for deps that don't support AndroidX yet
* Patches redux-offline, a dependency of aws-appsync, with postinstall script (thanks @nparashuram!)
* Upstream PR to redux offline here: https://github.com/redux-offline/redux-offline/pull/333

### CODE REVIEW

File: android/app/build.gradle [hunk 1]
```diff
@@ -186,3 +202,5 @@ task copyDownloadableDepsToLibs(type: Copy) {
     from configurations.compile
     into 'libs'
 }
+
+ //apply from: file("../../node_modules/@react-native-community/cli-platform-android/native_modules.gradle"); applyNativeModulesAppBuildGradle(project)
```
Comment (ryanlntn) [inline android/app/build.gradle]: Opt out of autolinking for now

File: android/settings.gradle [hunk 2]
```diff
@@ -27,4 +28,6 @@ project(':react-native-screens').projectDir = new File(rootProject.projectDir, '
 include ':react-native-splash-screen'
 project(':react-native-splash-screen').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-splash-screen/android')
 
+// apply from: file("../node_modules/@react-native-community/cli-platform-android/native_modules.gradle"); applyNativeModulesSettingsGradle(settings)
```
Comment (ryanlntn) [inline android/settings.gradle]: Opt out of autolinking for now

File: ios/Podfile [hunk 3]
```diff
@@ -1,25 +1,44 @@
...
+  pod 'appcenter-crashes', path: '../node_modules/appcenter-crashes/ios'
+  pod 'appcenter-push', path: '../node_modules/appcenter-push/ios'
+  pod 'react-native-netinfo', path: '../node_modules/@react-native-community/netinfo'
   pod 'RNCAsyncStorage', path: '../node_modules/@react-native-community/async-storage'
   pod 'RNGestureHandler', path: '../node_modules/react-native-gesture-handler'
   pod 'RNScreens', path: '../node_modules/react-native-screens'
+  pod 'RNDeviceInfo', path: '../node_modules/react-native-device-info'
 
-  pod 'RNDeviceInfo', :path => '../node_modules/react-native-device-info'
+  # use_native_modules!
```
Comment (ryanlntn) [inline ios/Podfile]: Opt out of autolinking for now

**Comment by robinheinze** [comment]
🎉 Nice work!

**Comment by jamonholmgren** [comment]
Ready to merge?

**Pull request merged**